einops/_backends.py

```diff
@@ -560,6 +578,12 @@ def layers(self):
         from .layers import keras
         return keras
 
+    def einsum(self, pattern, *x):
+        return self.tf.vectorized_map(
```

Comment:
want to understand why it looks so strange in tf.keras

Reply:
I'm not sure if I was interpreting the symbolic backends correctly or not. 

Basically, this `einsum` assumes the `x` tensors have a leading batch axis, which are assumed to not be specified in the pattern. I assumed this because the `create_symbol` method specifies the shape as a batch shape, rather than an absolute shape. Is that correct, or should it assume the pattern also specifies the batch axis?

Reply:
Actually, I think my implementation has a potential issue: if one symbol is batched, and one symbol is not (like a weight matrix).

Reply:
What do you think the correct strategy is here? Should I avoid adding `einsum` for keras, since it is technically a `layer=True` backend?

Reply:
layer=True just refers to providing layers, it should not be related to any batch variables, and patterns should include batch variables. Anyway, I forgot keras now just a redirection to TF layers, so just excluded this part